tests/cli/plugin/plugin_tests.py

```diff
@@ -23,7 +23,8 @@ def setUpClass(cls):
         TnsTest.setUpClass()
         Tns.create(cls.app_name, app_data=Apps.MIN_JS, update=False)
         Tns.platform_add_android(cls.app_name, framework_path=Settings.Android.FRAMEWORK_PATH)
-        Tns.platform_add_ios(cls.app_name, framework_path=Settings.IOS.FRAMEWORK_PATH)
+        if Platform is Platform.IOS:
```

Comment:
Shouldn't this be:  if Settings.HOST_OS is OSType.OSX:
Only when tests run on mac -> add platform ios